gateway-ha/src/test/java/io/trino/gateway/ha/HaGatewayTestUtils.java

```diff
@@ -159,6 +142,29 @@ public static void setUpBackend(
                         .build();
         Response response = httpClient.newCall(request).execute();
         assertThat(response.isSuccessful()).isTrue();
+        TrinoHealthStateType newClusterHealthState = TrinoHealthStateType.PENDING;
+        long startTime = System.currentTimeMillis();
+        // pull cluster health states for 10 seconds
+        // It should be enough as the healthcheck is run every second
+        int timeout = 10 * 1000;
+        while (newClusterHealthState != TrinoHealthStateType.HEALTHY && (System.currentTimeMillis() - startTime) < timeout) {
+            // check the state of newly added cluster every second
+            request = new Request.Builder()
+                    .url(String.format("http://localhost:%s/api/public/backends/%s/state", routerPort, name))
+                    .get()
+                    .build();
+            response = httpClient.newCall(request).execute();
```

Comment:
No need to avoid reusing `request` and `response` variables.

Reply:
no need or need?